pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -248,6 +248,31 @@ func fixMachineDriverLibvirtInstalled() (bool, error) {
 	return true, nil
 }
 
+func checkOldMachineDriverLibvirtInstalled() (bool, error) {
```

Comment:
@cfergeau Can we add comment on both the functions that this we should remove/revert after 2-3 release since after that no one using older version of crc due to cert issues.

Reply:
This will not cover someone who briefly tested crc now, and whose next test will be in 6 months. but I agree we can remove the test in a while.